Dockerfile

```diff
@@ -1,44 +1,94 @@
-FROM python:3.13-alpine3.21
+FROM node:20.19-alpine AS node-deps
+WORKDIR /src/recipes/vue3
 
-#Install all dependencies.
-RUN apk add --no-cache postgresql-libs postgresql-client gettext zlib libjpeg libwebp libxml2-dev libxslt-dev openldap git
+COPY --link ./vue3/package.json yarn.lock ./
 
-#Print all logs without buffering it.
-ENV PYTHONUNBUFFERED 1
+# TODO: use --frozen-lockfile
+# RUN yarn install --frozen-lockfile
+RUN yarn install
```

Comment:
Adding back `--frozen-lockfile` will be a requirement before merge, but uncommenting for now while cleaning up and moving forward